tests/unit/test_faultydata.py

```diff
@@ -96,7 +96,7 @@ class TestDuplicateCallSetId(FaultyVariantDataTest):
     """
     localIds = ["duplicated_sampleid"]
 
-    @unittest.skipIf(protocol.version.startswith("0.6"), "")
```

Comment:
We should still export ``protocol.version``. Nothing outside of the ``protocol`` module should have to interact with the generated modules in the ``proto`` hierarchy.